backend/src/routes/v2/model/postRequestExport.ts

```diff
@@ -0,0 +1,65 @@
+import bodyParser from 'body-parser'
+import { Request, Response } from 'express'
+import { z } from 'zod'
+
+import { AuditInfo } from '../../../connectors/audit/Base.js'
+import audit from '../../../connectors/audit/index.js'
+import { exportModel } from '../../../services/mirroredModel.js'
+import { registerPath } from '../../../services/specification.js'
+import { parse } from '../../../utils/validate.js'
+
+export const postRequestExportSchema = z.object({
+  params: z.object({
+    modelId: z.string(),
+  }),
+  body: z.object({
+    releases: z
+      .array(z.string())
+      .optional()
+      .openapi({ example: ['0.0.1', '0.0.2'] }),
+    disclaimerAgreement: z.boolean(),
+  }),
+})
+
+registerPath({
+  method: 'post',
+  path: '/api/v2/model/:modelId/export',
```

Comment:
I think we should have separate concepts of 'exporting', where we give the user a zip / pdf / etc and 'mirroring', where we transfer the model ourselves to another location.